src/main/java/uk/gov/pay/publicauth/dao/AuthTokenDao.java

```diff
@@ -82,7 +102,14 @@ public void storeToken(String tokenHash, String randomTokenLink, String accountI
                         .map(StringMapper.FIRST)
                         .first()));
     }
-
+    public Optional<String> revokeSingleTokenByExternalId(String accountExternalId, String tokenLink) {
+        return Optional.ofNullable(jdbi.withHandle(handle ->
+                handle.createQuery("UPDATE tokens SET revoked=(now() at time zone 'utc') WHERE account_external_id=:accountId AND token_link=:tokenLink AND revoked IS NULL RETURNING to_char(revoked,'DD Mon YYYY')")
+                        .bind("accountId", accountExternalId)
+                        .bind("tokenLink", tokenLink)
+                        .map(StringMapper.FIRST)
```

Comment:
StringMapper is deprecated can we use StringColumnMapper instead. Maybe change for revokeSingleToken above as well. Guess this is where it is copied from.